frontend/wiremock/flashcards-ui.md

```diff
@@ -0,0 +1,14 @@
+## Wiremock Design for Flashcards UI
+
+Excalidraw sketches for the UI design of the Flashcards app.
+
+### Overview pages
+
+This section contains the overview pages of the Flashcards app.
+
+#### Categories overview
+[Categories Overview Wiremock in Excalidraw](https://excalidraw.com/#json=8q6tI06UBIVsuKY4Yt-uG,QLPctNEjYY5RioPizpnpNA)
+
+#### Cards overview
+[Cards Overview Wiremock in Excalidraw](https://excalidraw.com/#json=LUwehQauaSV9Bz3vyioen,anF1csSMiiDYQoWSJQ9Yuw)
```

Comment:
Card Overview might benefit from a clear hint how to return to the Categories Overview, like a close icon in the corner.